core-web/libs/portlets/dot-ema/src/lib/components/edit-ema-persona-selector/edit-ema-persona-selector.component.scss

```diff
@@ -0,0 +1,25 @@
+@use "variables" as *;
+
+.edit-ema-persona-selector__persona {
+    display: flex;
+    align-items: center;
+    gap: $spacing-2;
+}
+
+::ng-deep {
+    .p-overlaypanel.edit-ema-persona-selector.p-component {
+        .p-overlaypanel-content {
+            padding: 0;
+
+            .p-listbox.p-component {
+                border: none;
+                border-radius: $border-radius-sm;
+
+                .p-listbox-header {
+                    border-top-left-radius: $border-radius-sm;
+                    border-top-right-radius: $border-radius-sm;
+                }
+            }
+        }
```

Comment:
Should we move this to global?

Reply:
Yes, because the listbox doesn't have styles matching our current look and feel